extension/lib/bigcommerce/JwtFactory.js

```diff
@@ -0,0 +1,59 @@
+const jwt = require('jwt-simple')
+const randomBytes = require('randombytes')
+
+/**
+ * @property {string} _clientId
+ * @property {string} _storeHash
+ * @property {string} _clientSecret
+ */
+class JwtFactory {
```

Comment:
Try naming this class AuthRepository instead, I think it will give more context to it.

Reply:
Done